packages/theme/getters/productGetters.ts

```diff
@@ -6,13 +6,11 @@ import {
   ProductGetters as ProductGettersBase,
 } from '@vue-storefront/core';
 import {
-  BundleProduct,
+  htmlDecode, BundleProduct,
```

Comment:
```suggestion
  BundleProduct,
```
Same as in previous comment - please use the `theme` version (import {htmlDecode} from '../helpers/htmlDecoder')

Reply:
Hmm, I can't find htmlDecoder in theme.

Reply:
Right, misread on my side